### PR TITLE
Fix minor things in the README and backend install instructions

### PR DESCRIPTION
**Pull request by raphj**
Hello,

Here are some minor fixes to the README and the installation instructions for the backend.

Best!
Raphaël

## Checkpoints

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

### CODE REVIEW

**Comment by Skylsmoi** [comment]
I've checked every checkpoints since the MR only does documentation changes.

**Comment by Skylsmoi** [comment]
@raphj Thanks a lot for these fixes !

File: backend/README.md [hunk 1]
```diff
@@ -116,28 +113,26 @@ Stamp current version of database to last (useful for migration):
...
 
-    # install uwsgi on debian 9
-    sudo apt install uwsgi uwsgi-plugin-python3
+    sudo pip3 install uwsgi
 
 #### All in terminal way ####
 
 
-Run all web services with uwsgi
+Run all web services with UWSGI:
```
Comment (lebouquetin) [inline backend/README.md]: @Skylsmoi @raphj the project uwsgi is uWSGI, not UWSGI

Comment (raphj) [reply backend/README.md]: Indeed, I messed up this occurrence. Sorry for that. I pushed a commit to my branch to fix this, don't hesitate to merge it. I can also open a new pull request and I don't mind if you fix this without using my commit.

Here is the patch file if it is more convenient for you (`git am < 0001-Fix-UWSGI-uWSGI-in-the-backend-installation-doc.patch`). GitHub does not let me upload the patch as a file.

```
From 137db6107400b62cbe685027ebe2b1809b79eb19 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Rapha=C3=ABl=20Jakse?= <raphael.github@jakse.fr>
Date: Fri, 10 Jan 2020 08:27:28 +0100
Subject: [PATCH] =?UTF-8?q?Fix=20UWSGI=20=E2=86=92=20uWSGI=20in=20the=20ba?=
 =?UTF-8?q?ckend=20installation=20doc?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Raphaël Jakse <raphael.github@jakse.fr>
---
 backend/README.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/backend/README.md b/backend/README.md
index 9e86056d..2a6ed3c0 100644
--- a/backend/README.md
+++ b/backend/README.md
@@ -132,7 +132,7 @@ Or with PIP:
 #### All in terminal way ####
 
 
-Run all web services with UWSGI:
+Run all web services with uWSGI:
 
     ## UWSGI SERVICES
     # set tracim_conf_file path
-- 
2.24.1
```

**Pull request merged**